src/current/v23.1/secure-cockroachdb-kubernetes.md

```diff
@@ -528,6 +528,48 @@ If you previously [authenticated with `cockroach cert`](#example-authenticating-
 
     The certificates will be rotated during the specified expiry windows.
 
+## Deploy cert-manager for mTLS
+
+The recommended approach for cluster authentication is to use cert-manger to sign certificates. cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.
+
+1. Make sure you have the latest version of cert-manger installed you will find it [here](https://cert-manager.io/docs/installation/).
+
+2. The first thing you will need to do is to create an `Issuer` these are the resources that represent certificate authorities and are able to sign certificates. Using cert-manger create an `Issuer` for signing self-signed CA certificate.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~yaml
+    apiVersion: cert-manager.io/v1
+    kind: Issuer
+    metadata:
+    name: cockroachdb
+    spec:
+    selfSigned: {}
+
+3. Apply this to your Kubernetes cluster using `kubectl`
+
+4. To enable `cert-manager` you will need to have the following settings in the values.yaml file.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~yaml
+    tls.certs.selfSigner.enabled: false
+    tls.certs.certManager: true
+    tls.certs.certManagerIssuer.kind: Issuer
+    tls.certs.certManagerIssuer.name: cockroachdb
+    ~~~
+
+- Setting `tls.certs.selfSigner.enabled` to `false` will disable the self signing certificates for cockroachdb.
+- By enabling the`tls.certs.certManager:` setting to true this will enable cert-manager for certificate signing.
+- Specify the ‘Issuer` kind here either `Issuer` or `ClusterIssuer` under the setting `certManagerIssuer.kind`.
+- `certManagerIssuer.name` Provide the Issuer you have created in previous step. In this example it was `cockroachdb`.
+
+5. Once the values.yaml file has been updated you can then deploy CockroachDB via the Helm Chart with the following command.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    helm install my-release --values {custom-values}.yaml cockroachdb/cockroachdb
+    ~~~
+
+
 ## Migration to self-signer
```

Comment:
Should this section be removed?

Reply:
I think that it should be removed and cert-manager adopted moving forward.